src/GafferImageUI/ImageGadget.cpp

```diff
@@ -75,6 +78,401 @@ using namespace GafferUI;
 using namespace GafferImage;
 using namespace GafferImageUI;
 
+namespace
+{
+
+// These supporting functions, along with RenderTexture itself, contain code copied from ViewportGadget and
+// rearranged. It's expressed in such a way that we could use it as a drop-in replacement for the code in
+// ViewportGadget if we wanted to avoid the duplication - but this would require exposing it more than we
+// are currently comfortable with.
+void renderUnitSquare( const IECoreGL::Shader::Parameter *pParameter, const IECoreGL::Shader::Parameter *uvParameter = nullptr )
+{
+	static float rectPBufferData[12] = { -1, -1, 0,  -1, 1, 0,  1, -1, 0,  1, 1, 0 };
+	static float rectUvBufferData[12] = { 0, 0,  0, 1,  1, 0,  1, 1 };
```

Comment:
Should the array length be 8 here?

Reply:
Fixed here:[5320f77](https://github.com/GafferHQ/gaffer/pull/6329/commits/5320f776de0d82d40b0e717a66580cf4d4a277e2)

And here: [989a141](https://github.com/GafferHQ/gaffer/pull/6329/commits/989a1410f25ac999b2af6859a6756af6c9484e60)